src/release.js

```diff
@@ -0,0 +1,59 @@
+const github = require('@actions/github');
+
+async function getLatestRelease(token) {
+  const octokit = github.getOctokit(token);
+  const { owner, repo } = github.context.repo;
+
+  const allReleasesResp = await octokit.request('GET /repos/{owner}/{repo}/releases', {
+    owner,
+    repo,
+  });
+
+  const latestRelease = (allReleasesResp && allReleasesResp.data && allReleasesResp.data.length)
+    ? allReleasesResp.data[0] : null;
+  if (!latestRelease) throw new Error('Cannot find the latest release');
+
+  return latestRelease;
+}
+
+async function getUnreleasedCommits(token, latestRelease, daysToIgnore) {
+  const octokit = github.getOctokit(token);
+  const { owner, repo } = github.context.repo;
+
+  // TODO: extract this
+  const allCommitsResp = await octokit.request('GET /repos/{owner}/{repo}/commits', {
+    owner,
+    repo,
+  });
+
+  if (!allCommitsResp || !allCommitsResp.data || !allCommitsResp.data.length) throw new Error('Error fetching commits');
+  if (!latestRelease || !latestRelease.created_at) throw new Error('Latest release doesnt have a created_at date');
```

Comment:
same thing as above, these conditions can probably be simplified if you can make assumptions about the return values of the octokit apis